docs/en/observability/whats-new.asciidoc

```diff
@@ -55,4 +54,18 @@ and we encourage you to provide feedback via the Discuss link provided within th
 [role="screenshot"]
 image::images/81-alerts.png[Improved alerts view and alert reason messages]
 
+[discrete]
+== APM Tail-based sampling now GA
+
+We are pleased to announce that tail-based sampling has been made generally available (GA) in APM.
+
+Tail-based sampling enables users to optimize their sampling behavior by configuring sampling policies that determine how traces will be sampled.
+
+Since sampling decisions are determined against policy criteria once a trace has completed, the risk of discarding important data is reduced.
+
+Tail-based sampling can be configured in APM Integration Settings:
```

Comment:
Should we link this line to https://www.elastic.co/guide/en/apm/guide/current/configure-tail-based-sampling.html?

Reply:
Yes, good call - will add to a couple of minor fixes and add the commit